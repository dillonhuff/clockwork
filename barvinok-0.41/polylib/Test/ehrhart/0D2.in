6 5
    1     0     0    -1    19 
    1     1     0     0    -1 
    1     0     1     0   -17 
    1    -1     0     0    19 
    1     0    -1     0    19 
    1     0     0     1     0 

0 5
