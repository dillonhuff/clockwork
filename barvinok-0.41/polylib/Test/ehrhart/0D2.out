---------------------------------------
Domain:
         - R + 19 >= 0
         P  -1 >= 0
         Q  -17 >= 0
         - P + 19 >= 0
         - Q + 19 >= 0
         R  >= 0


Ehrhart Polynomial:
1
